medusa/providers/torrent/html/archetorrent.py

```diff
@@ -0,0 +1,215 @@
+            'freeleech': freeleech,  # Only freeleech torrent - 0: off 1: no freeleech 2: Only freeleech
+            'lang': '0'  # Language - 0: off 1: English 2: French ....
+        }
+
+        for mode in search_strings:
+            log.debug('Search mode: {0}', mode)
+
+            for search_string in search_strings[mode]:
+
+                if mode != 'RSS':
+                    log.debug('Search string: {search}',
+                              {'search': search_string})
+
+                search_params['search'] = search_string
+                response = self.session.get(self.urls['search'], params=search_params)
+                if not response or not response.text:
+                    log.debug('No data returned from provider')
+                    continue
+
+                results += self.parse(response.text, mode, keywords=search_string)
+
+        return results
+
+    def parse(self, data, mode, **kwargs):
+        """
+        Parse search results for items.
+
+        :param data: The raw response from a search
+        :param mode: The current mode used to search, e.g. RSS.
+
+        :return: A list of items found
+        """
+        # Units
+        units = ['B', 'KB', 'MB', 'GB', 'TB', 'PB']
+
+        items = []
+
+        keywords = kwargs.pop('keywords', None)
+
+        with BS4Parser(data, 'html5lib') as html:
+            torrent_table = html.find(class_='ttable_headinner')
+            torrent_rows = torrent_table('tr') if torrent_table else []
+
+            # Continue only if at least one release is found
+            if len(torrent_rows) < 2:
+                log.debug('Data returned from provider does not contain any torrents')
+                return items
+
+            # Catégorie, Release, Date, DL, Size, C, S, L
+            labels = [label.get_text(strip=True) for label in torrent_rows[0]('th')]
+
+            for torrent in torrent_rows[1:]:
+                cells = torrent('td')
+                if len(cells) < len(labels):
+                    continue
+
+                try:
+                    torrent_id = re.search('id=([0-9]+)', cells[labels.index('Nom')].find('a')['href']).group(1)
+                    title = cells[labels.index('Nom')].get_text(strip=True)
+                    download_url = urljoin(self.urls['download'], '?id={0}&name={1}'.format(torrent_id, title))
+                    if not all([title, download_url]):
+                        continue
+
+                    # Chop off tracker/channel prefix or we cannot parse the result!
+                    if mode != 'RSS' and keywords:
+                        show_name_first_word = re.search(r'^[^ .]+', keywords).group()
+                        if not title.startswith(show_name_first_word):
+                            title = re.sub(r'.*(' + show_name_first_word + '.*)', r'\1', title)
+
+                    seeders = try_int(cells[labels.index('S')].get_text(strip=True))
```

Comment:
We don't really like using try_int anymore, unless there is really no other options. And generally there always are.
Please replace by int().